cfp.yml

```diff
@@ -0,0 +1,25 @@
+"PLDI":
+  - year: 2025
+  - url: "[PLDI'25](https://conf.researchr.org/series/pldi)"
```

Comment:
@uchitsa doesn't look like URL to me, more like Markdown. I believe, it would be better to have only URL here and generate Markdown when the page is generated